File: 151001/Shlyk/SpringCalculator/spring-calculator-rest/src/main/java/by/bsuir/poit/dc/rest/api/dto/response/NoteDto.java
package by.bsuir.poit.dc.rest.api.dto.response;

import com.baidu.bjf.remoting.protobuf.annotation.ProtobufClass;
import lombok.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@ProtobufClass
public class NoteDto {
    private long id;
    private String content;
    private long newsId;
}

File: 151004/Pyshny/RV/RV/Views/DTO/NoteDTO.cs
using System.ComponentModel.DataAnnotations;

namespace RV.Views.DTO
{
    public class NoteDTO
    {
        [Required]
        public int id { get; set; }
        [Required]
        public int newsId { get; set; }
        [Required]
        public string content { get; set; }
    }
}
